docs/release-procedure.md

```diff
@@ -106,12 +106,12 @@ Once the voting is passed, move the release artefacts to https://dist.apache.org
 Once moved to this space, the content will be automatically synced to https://downloads.apache.org/incubator/yunikorn/ which can be used as the final location for release files.
 Read more for [location of files on main server](https://infra.apache.org/mirrors#location).
 
-Publish an announcement blog to https://blogs.apache.org/yunikorn/, update the web-site with corresponding
-release notes, download links.
+## Release Docker images
+
+Push the latest docker images to docker hub.
```

Comment:
I think we should mention the tagged version here. Make sure the docker image is built on the specific SHA.

Reply:
Added.